src/mode/modeHandler.ts

```diff
@@ -124,6 +121,41 @@ export class VimState {
   public currentFullAction: string[] = [];
 
   /**
+   * Getters and setters for changing global state
+   */
+  public get searchStatePrevious(): SearchState[]{
```

Comment:
So instead of saying `vimState.searchState` they would say `vimState.globalState.searchState`?